enterprise/frontend/src/metabase-enterprise/tools/containers/ErrorDetail.jsx

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import { t } from "ttag";
+import PropTypes from "prop-types";
+import { getIn } from "icepick";
+
+import { formatColumn, formatValue } from "metabase/lib/formatting";
+import { CardApi } from "metabase/services";
+import Button from "metabase/components/Button";
+import Link from "metabase/components/Link";
+import Question from "metabase-lib/lib/Question";
+import { QuestionResultLoader } from "metabase/containers/QuestionResultLoader";
+import { columnNameToUrl } from "../../audit_app/lib/mode";
+
+const CARD_ID_ROW_IDX = 0;
+const ErrorDrill = ({ clicked }) => {
+  if (!clicked) {
+    return [];
+  }
+
+  const cardId = clicked.origin.row[CARD_ID_ROW_IDX];
+
+  return [
+    {
+      name: "detail",
+      title: `View this`,
+      default: true,
+      url() {
+        return `/admin/tools/errors/${cardId}`;
+      },
+    },
+  ];
+};
+
+export const ErrorMode = {
```

Comment:
What do you think about storing this mode in a separate file for better searchability?